routes/StipendApplicationRoute.js

```diff
@@ -13,6 +13,7 @@ const router = require("express").Router();
 // router.use(isWindowOpen)
 
 router.post("/apply", isAuthenticated, createStipendApplication);
+// I think they should be authenticated for this
```

Comment:
Yes they will be. It's part of what I'll be working on for February release @Chiemezuo